docs: add callout for multitenancy workshop (#3460)
cube-js/cube.js: docs/content/Configuration/Multitenancy-Setup.mdx

menuOrder: 4
---

> <!-- prettier-ignore-start -->
> [[info |]]
> | The Cube multitenancy workshop is on October 27th at 9-10:30 am PST! If you
> | want to learn why you need multitenancy, how to get started,
> | recipes, you will want to attend this event 😀 <br/> You can register
> | for the workshop at [the event
> | page](https://cube.dev/events/multitenancy/).
> <!-- prettier-ignore-end -->
> 

Reviewer: ```suggestion
<InfoBox>
The Cube multitenancy workshop is on October 27th at 9-10:30 am PST! If you want to learn why you need multitenancy, how to get started, recipes and more, you will want to attend this event 😀 

You can register for the workshop at [the event page](https://cube.dev/events/multitenancy/).
</InfoBox>
```
